grape-active_model_serializers.gemspec

```diff
@@ -18,6 +18,7 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency 'grape', '>= 0.8.0'
   gem.add_dependency 'active_model_serializers', '>= 0.10.0'
+  gem.add_dependency 'multi_json', '~> 1.0'
```

Comment:
I don't think we want this. Grape has a way to format JSON regardless of the library used, you can call it with `::Grape::Json.dump`.

Reply:
It's [here](https://github.com/ruby-grape/grape/blob/master/lib/grape/util/json.rb)

Reply:
Then do you want to change the version of grape depended on?

Reply:
Or should we implement the same thing?

Reply:
This gem depends on grape, so you should be able to use it, I am not sure what the problem is.

Reply:
So, this one should still be removed, it's unnecessary in all Grape versions since it's either coming via Grape or is not used..